letsencrypt-auto

```diff
@@ -14,7 +14,19 @@ VENV_PATH=${VENV_PATH:-"$XDG_DATA_HOME/$VENV_NAME"}
 VENV_BIN=${VENV_PATH}/bin
 
 if test "`id -u`" -ne "0" ; then
-  SUDO=sudo
+  if type sudo 1>/dev/null 2>&1; then
```

Comment:
`command -v` actually is a [POSIX semi-standard](http://pubs.opengroup.org/onlinepubs/009696899/utilities/command.html). 

I used `which` for python versoin-wrangling elsewhere in the shell; we should probably just use one of these...